recipes/catch2/3.x.x/conanfile.py

```diff
@@ -17,111 +18,109 @@ class Catch2Conan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     settings = "os", "arch", "compiler", "build_type"
     options = {
+        "shared": [True, False],
         "fPIC": [True, False],
         "with_prefix": [True, False],
-        "default_reporter": "ANY",
```

Comment:
This option still exists 😕 https://github.com/catchorg/Catch2/blob/c0e582e65955f84312548bcee36b962a013f262f/CMake/CatchConfigOptions.cmake#L72

https://github.com/catchorg/Catch2/blob/c0e582e65955f84312548bcee36b962a013f262f/docs/configuration.md#default-reporter

I think this should be kept

Reply:
indeed